src/Utils/TerminalUtils.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drush\Utils;
+
+class TerminalUtils
+{
+    public static function stdinIsTerminal($default = true)
```

Comment:
Seems like isTty() would be more consistent name. 

Could so with doxygen here if you have time.